### PR TITLE
Update tags orgs

### LINKED ISSUE #4: Update tags_organisations.json
We are currently using `primary_3` (indicating primary user, who has id = 3) to test primary user views with.
The issue is that the organisation (id=1) attached to `primary_3`, does not share a tag in common with any of the other organisations, so when testing for suggested matches ([as can be seen in this PR](https://github.com/emfoundation/ce100-app/pull/506)) we are returned with no matches.

We need to update tags_organisations.json so that organisation (id = 1) can have suggested matches.

### PR DESCRIPTION
**Pull request by mk4111**
Closes #4

Updates tags-orgs so that the organisation (of the primary user - id= 3 - that we use for testing) can match with another organisation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "c100-mock-data",
-  "version": "1.0.0",
+  "name": "ce100-mock-data",
+  "version": "1.0.3",
```
Comment (jay-meister) [inline package.json]: I think version number should be 1.0.4

Comment (mk4111) [reply package.json]: Yep!!

**Comment by jay-meister** [comment]
done @SimonLab over to you

**Review by SimonLab — APPROVED** [review]
LGTM

**Pull request merged**